front/app/api/idea_images/keys.ts

```diff
@@ -9,11 +9,11 @@ const ideaImagesKeys = {
     { ...baseKey, operation: 'list', parameters: { ideaId } },
   ],
   items: () => [{ ...baseKey, operation: 'item' }],
-  item: ({ ideaId, imageId }: { ideaId: string; imageId?: string }) => [
+  item: ({ imageId }: { imageId?: string }) => [
```

Comment:
Just a question - in some cases we keep the more descriptive parameter here (E.g. "imageId") and in others we use just "id":
```
item: ({ id }: { id: string }) => [
``` 
Is one form preferred over the other? :)

Reply:
Wherever the parameter was named something other than `id` I kept the name so that I limit the amount of changes needed. It's not really that important what it's called, as long as it's passed as `id` in the parameters. Therefore, I wouldn't say one approach is preferred over the other, it was just a bit of an opportunistic way to deal with it on my end.

Reply:
Ok for sure :) Thanks!!